File: URI/1 - INICIANTE/Python/1172 - SubstituicaoEmVetor1.py
vetor = []

for x in range(0, 10):
    y = int(input())
    if y > 0:
        vetor.append(y)
    else:
        vetor.append(1)
for x in range(0, 10):
    print('X[{}] = {}'.format(x,vetor[x]))

File: URI/1 - INICIANTE/C/1172 - SubstituicaoEmVetor1.c
#include <stdio.h>

int main()
{
	int x, vetor[10], y;
	
	for(x = 0; x < 10; x++)
	{
		scanf(" %d", &y);
		if(y > 0){
			vetor[x] = y;
		}else{
			vetor[x] = 1;
		}
	}
	for(x = 0; x < 10; x++){
		printf("X[%d] = %d\n", x, vetor[x]);
	}
	
	return 0;
}
